src/main/java/org/openstreetmap/atlas/generator/creator/RawAtlasCreator.java

```diff
@@ -0,0 +1,378 @@
+import org.openstreetmap.atlas.geography.atlas.AtlasResourceLoader;
+import org.openstreetmap.atlas.geography.atlas.dynamic.DynamicAtlas;
+import org.openstreetmap.atlas.geography.atlas.packed.PackedAtlas;
+import org.openstreetmap.atlas.geography.atlas.packed.PackedAtlas.AtlasSerializationFormat;
+import org.openstreetmap.atlas.geography.atlas.pbf.AtlasLoadingOption;
+import org.openstreetmap.atlas.geography.atlas.raw.creation.RawAtlasGenerator;
+import org.openstreetmap.atlas.geography.atlas.raw.sectioning.WaySectionProcessor;
+import org.openstreetmap.atlas.geography.atlas.raw.slicing.RawAtlasCountrySlicer;
+import org.openstreetmap.atlas.geography.boundary.CountryBoundaryMap;
+import org.openstreetmap.atlas.geography.sharding.CountryShard;
+import org.openstreetmap.atlas.geography.sharding.Shard;
+import org.openstreetmap.atlas.geography.sharding.Sharding;
+import org.openstreetmap.atlas.geography.sharding.SlippyTile;
+import org.openstreetmap.atlas.streaming.resource.File;
+import org.openstreetmap.atlas.streaming.resource.FileSuffix;
+import org.openstreetmap.atlas.utilities.collections.Maps;
+import org.openstreetmap.atlas.utilities.conversion.StringConverter;
+import org.openstreetmap.atlas.utilities.runtime.Command;
+import org.openstreetmap.atlas.utilities.runtime.CommandMap;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * A raw atlas compatible version of the {@link AtlasCreator} command. This tool allows the user to
+ * replicate raw flow {@link AtlasGenerator} conditions on a certain shard. It is
+ * {@link DynamicAtlas} enabled, and will create sliced shards for expansion where necessary.
+ *
+ * @author lcram
+ */
+public class RawAtlasCreator extends Command
+{
+    /**
+     * Represents the different possible stages (or flavors) in the raw atlas flow.
+     *
+     * @author lcram
+     */
+    private enum RawAtlasFlavor
+    {
+        RawAtlas("raw"),
+        SlicedRawAtlas("sliced"),
+        SectionedRawAtlas("sectioned");
+
+        private final String flavorString;
+
+        public static RawAtlasFlavor flavorStringToRawAtlasFlavor(final String string)
+        {
+            for (final RawAtlasFlavor flavor : RawAtlasFlavor.values())
+            {
+                if (flavor.toString().equalsIgnoreCase(string))
+                {
+                    return flavor;
+                }
+            }
+            throw new CoreException("Invalid RawAtlasFlavor {}", string);
+        }
+
+        RawAtlasFlavor(final String flavorString)
+        {
+            this.flavorString = flavorString;
+        }
+
+        @Override
+        public String toString()
+        {
+            return this.flavorString;
+        }
+    }
+
+    private static final Logger logger = LoggerFactory.getLogger(RawAtlasCreator.class);
+    private static final String DEFAULT_CACHE_NAME = "__RawAtlasCreator_slicedAtlasCache__";
+
+    /*
+     * A path to a country boundary map file
```

Comment:
The javadoc-style comments are usually reserved for methods. For consistency, I'd use `//` here instead.

Reply:
Had not noticed all those are one-star... All good :)